src/store/metadata/oxigraph/store.py

```diff
@@ -110,7 +118,60 @@ def get_edition(
         """
         Gets a specific edition of a specific dataset
         """
-        raise NotImplementedError
+        # Populate the graph from the database
+        graph = self.db
+
+        # Use the construct wrappers to pull the raw RDF triples
+        # (as one rdflib.Graph() for each function) and add them
+        # together to create a single Graph of the
+        # data we need.
+        result: Graph = (
+            construct_edition_core(graph, dataset_id, edition_id)
+            + construct_edition_contact_point(graph, dataset_id, edition_id)
+            + construct_edition_topics(graph, dataset_id, edition_id)
+            + construct_edition_keywords(graph, dataset_id, edition_id)
+            + construct_edition_temporal_coverage(graph, dataset_id, edition_id)
+            + construct_edition_table_schema(graph, dataset_id, edition_id)
+            + construct_edition_versions(graph, dataset_id, edition_id)
+        )
+
+        # Serialize the graph into jsonld
+        data = json.loads(result.serialize(format="json-ld"))
+
+        # Use a context file to shape our jsonld, removing long form references
+        data = jsonld.flatten(
+            data, {"@context": constants.CONTEXT, "@type": "dcat:Dataset"}
+        )
+
+        edition_graph = _get_single_graph_for_field(data, "@type")
+        contact_point_graph = _get_single_graph_for_field(data, "vcard:fn")
+        temporal_coverage_graph = _get_single_graph_for_field(data, "dcat:endDate")
+        columns_graph = [x for x in data["@graph"] if "datatype" in x.keys()]
+
+        # Populate editions_graph.table_schema.columns with column definitions (without @id) and delete editions_graph.table_schema blank node @id
+        for column in columns_graph:
+            del column["@id"]
+        edition_graph["table_schema"]["columns"] = columns_graph
+        del edition_graph["table_schema"]["@id"]
+
+        edition_graph["contact_point"] = {
+            "name": contact_point_graph["vcard:fn"]["@value"],
+            "email": contact_point_graph["vcard:hasEmail"],
+        }
+        edition_graph["temporal_coverage"] = {
+            "start": temporal_coverage_graph["dcat:startDate"]["@value"],
+            "end": temporal_coverage_graph["dcat:endDate"]["@value"],
+        }
+
+        # Add `issued` and `modified` fields to each version in `versions`
```

Comment:
comment needs a quick update (or delete if you'd prefer), its not really doing this anymore